src/app/code/community/Zendesk/Zendesk/Model/Observer.php

```diff
@@ -109,4 +110,97 @@ public function addTicketButton(Varien_Event_Observer $event)
             ));
         }
     }
+    
+    public function changeIdentity(Varien_Event_Observer $event)
+    {
+        if(!Mage::getStoreConfig('zendesk/general/customer_sync'))
+            return;
+        
+        $user = null;
+        $customer = $event->getCustomer();
+        $email = $customer->getEmail();
+        $orig_email = $customer->getOrigData();
+        $orig_email = $orig_email['email'];
+        
+        //Get Customer Group
+        $group_id = $customer->getGroupId();
+        $group = Mage::getModel('customer/group')->load($group_id);
+        
+        //Get Customer Last Login Date
+        $log_customer = Mage::getModel('log/customer')->loadByCustomer($customer);  
+        
+        //Get Customer Sales Statistics
+        $order_totals = Mage::getResourceModel('sales/order_collection');
+        $lifetime_sale = 0;
+        $average_sale = 0;
+        
+        if (is_object($order_totals)) {
+            $order_totals
+            ->addFieldToSelect('*')
+            ->addFieldToFilter('customer_id', $customer->getId())
+            ->addFieldToFilter('status', Mage_Sales_Model_Order::STATE_COMPLETE)
+            ->addAttributeToSelect('grand_total')
+            ->getColumnValues('grand_total');
+            
+            $sum = 0;
+            foreach ($order_totals as $total) {
+                if (isset($total['grand_total']))
+                    $sum += (float)$total['grand_total'];
+            }
+            
+            $lifetime_sale = Mage::helper('core')->currency($sum, true, false);
+            $average_sale = Mage::helper('core')->currency($sum / count($order_totals), true, false);
+        }
+        
+        $info['user'] = array(
+                "name"          =>  $customer->getFirstname() . " " . $customer->getLastname(),
+                "email"         =>  $email,
+                "user_fields"       =>  array(
+                    "group"         =>  $group->getCode(),
+                    "name"          =>  $customer->getFirstname() . " " . $customer->getLastname(),
+                    "id"            =>  $customer->getId(),
+                    "logged_in"     =>  date("Y-m-d\TH:i:s\Z",strtotime($log_customer->getLoginAt())),
```

Comment:
If a customer hasn't logged in before this returns `1970-01-01T00:00:00Z`. It should just be empty if the customer hasn't logged in.